elements/rh-navigation-secondary/rh-navigation-secondary.ts

```diff
@@ -389,11 +386,8 @@ export class RhNavigationSecondary extends LitElement {
     this.removeAttribute('role');
     // remove aria-labelledby from slotted `<ul>` on upgrade
     this.querySelector(':is([slot="nav"]):is(ul)')?.removeAttribute('aria-labelledby');
-    // transfer the aria-label to the shadow <nav>
-    if (this.hasAttribute('aria-label')) {
-      this.#label = this.getAttribute('aria-label') ?? 'secondary';
-      this.removeAttribute('aria-label');
-    }
+    // if the accessibleLabel attr is undefined, check aria-label if undefined use default
+    this.#internals.ariaLabel = this.getAttribute('aria-label') ?? 'secondary';
```

Comment:
We shouldn't need this. Internals are defaults which can be overridden by host attrs

At most, we may choose to provide an `accessible-label` attr, for the sake of congruency with other elements. That attr would set the internals property, but would still be overridden by aria-label on host

Reply:
makes sense, and fixed, thanks